pkg/synthetictests/networking.go

```diff
@@ -70,6 +70,14 @@ func testPodSandboxCreation(events monitorapi.Intervals) []*junitapi.JUnitTestCa
 			// See https://github.com/openshift/origin/blob/93eb467cc8d293ba977549b05ae2e4b818c64327/test/extended/networking/whereabouts.go#L52
 			continue
 		}
+		if strings.Contains(event.Locator, "pod/simpletest-rc-to-be-deleted") &&
+			strings.Contains(event.Message, "failed to create pod network sandbox k8s_simpletest-rc-to-be-deleted") &&
+			strings.HasSuffix(event.Message, " not found") {
+			// This failed to create a sandbox case is expected due to the ovnkube-msater pod trying to bring up the logical port
+			// for a pod that is in the process of being deleted.
+			// See https://github.com/openshift/origin/blob/36ddf0503bbb9f7adf9fe3481f85c856ec0a2f31/vendor/k8s.io/kubernetes/test/e2e/apimachinery/garbage_collector.go#L739
+			continue
+		}
 		deletionTime := getPodDeletionTime(eventsForPods[event.Locator], event.Locator)
 		if deletionTime == nil {
```

Comment:
ha!  This test is backwards isn't it?  See usage on old line 91

Reply:
hm... does that mean we can just change `deletionTime == nil` to `deletionTime != nil` and remove my code?

Reply:
In the events data for this job, I see:

```
{
  "level": "Warning",
  "locator": "ns/e2e-gc-6231 pod/simpletest-rc-to-be-deleted-7kz7f node/ci-op-xy9lvk43-de73b-lnh2s-worker-a-bjsmj",
  "message": "reason/FailedCreatePodSandBox Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create pod network sandbox k8s_simpletest-rc-to-be-deleted-7kz7f_e2e-gc-6231_53d52faf-39f9-499a-87b1-331bff8106b1_0(2e6eb6a27824046e9df7447a048524648dc2828ca0d1f07bf1a49cefdf31fa40): error adding pod e2e-gc-6231_simpletest-rc-to-be-deleted-7kz7f to CNI network \"multus-cni-network\": plugin type=\"multus\" name=\"multus-cni-network\" failed (add): Multus: [e2e-gc-6231/simpletest-rc-to-be-deleted-7kz7f/53d52faf-39f9-499a-87b1-331bff8106b1]: error getting pod: pods \"simpletest-rc-to-be-deleted-7kz7f\" not found",
  "from": "2022-04-05T08:12:11Z",
  "to": "2022-04-05T08:12:11Z"
}
{
  "level": "Info",
  "locator": "ns/e2e-gc-6231 pod/simpletest-rc-to-be-deleted-7kz7f node/ci-op-xy9lvk43-de73b-lnh2s-worker-a-bjsmj uid/53d52faf-39f9-499a-87b1-331bff8106b1",
  "message": "reason/ForceDelete mirrored/false",
  "from": "2022-04-05T08:12:11Z",
  "to": "2022-04-05T08:12:11Z"
}
{
  "level": "Info",
  "locator": "ns/e2e-gc-6231 pod/simpletest-rc-to-be-deleted-7kz7f node/ci-op-xy9lvk43-de73b-lnh2s-worker-a-bjsmj uid/53d52faf-39f9-499a-87b1-331bff8106b1",
  "message": "reason/Deleted ",
  "from": "2022-04-05T08:12:11Z",
  "to": "2022-04-05T08:12:11Z"
}
```

the event.Locator for the `FailedCreatePodSandBox` event is not the same as the one(s) for the `reason/Delete`.  So that code will not find the `reason/Delete` event for that pod.

Reply:
I'm having trouble understanding what you mean by backwards.